apps/odyssey-stats/src/widget/mini-chart.jsx

```diff
@@ -0,0 +1,68 @@
+import { translate } from 'i18n-calypso';
+import moment from 'moment';
+import { useState } from 'react';
+import Intervals from 'calypso/blocks/stats-navigation/intervals';
+import Chart from 'calypso/components/chart';
+import Legend from 'calypso/components/chart/legend';
+import { buildChartData } from 'calypso/my-sites/stats/stats-chart-tabs/utility';
+import StatsModulePlaceholder from 'calypso/my-sites/stats/stats-module/placeholder';
+import useVisitsQuery from '../hooks/use-visits-query';
+
+import './mini-chart.scss';
+
+const CHART_VIEWS = {
+	attr: 'views',
+	legendOptions: [ 'visitors' ],
+	label: translate( 'Views', { context: 'noun' } ),
+};
+
+const CHART_VISITORS = {
+	attr: 'visitors',
+	label: translate( 'Visitors', { context: 'noun' } ),
+};
+
+const CHARTS = [ CHART_VIEWS, CHART_VISITORS ];
+
+const MiniChart = ( { siteId, quantity = 7, gmtOffset, odysseyStatsBaseUrl } ) => {
```

Comment:
The hover tooltip is missing the month and day formatting:

<img width="270" alt="image" src="https://user-images.githubusercontent.com/4044428/230211651-0f41e18f-ca1c-4dff-ace0-06cc3b760b9a.png">

Reply:
Good catch! This is related to https://github.com/Automattic/wp-calypso/pull/75255 and I'll address in a follow up PR. 👍